plugins/rts54hub/fu-rts54hub-rtd21xx-mergeinfo.c

```diff
@@ -0,0 +1,481 @@
+/*
+ * Copyright 2021 Realtek Corporation
+ * Copyright 2021 Ricky Wu <ricky_wu@realtek.com> <spring1527@gmail.com>
+ * Copyright 2021 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ */
+
+#include "config.h"
+
+#include "fu-rts54hub-device.h"
+#include "fu-rts54hub-rtd21xx-mergeinfo.h"
+
+struct _FuRts54hubRtd21xxMergeinfo {
+	FuRts54hubRtd21xxDevice parent_instance;
+};
+
+G_DEFINE_TYPE(FuRts54hubRtd21xxMergeinfo,
+	      fu_rts54hub_rtd21xx_mergeinfo,
+	      FU_TYPE_RTS54HUB_RTD21XX_DEVICE)
+
+#define DEBUG_TARGET_ADDR	    0x6A
+#define DDCCI_TARGET_ADDR	    0x6E
+#define CHANGE_TO_DDCCI_MODE_OPCODE 0x71
+#define DDCCI_COMM_SUB_ADDR	    0x71
+#define CHECK_ACK_SUB_ADDR	    0x23
+#define VERSION_NUMBER_COUNT	    4
+#define DDCCI_COMM_FIRST_OPCODE	    0x77
+#define DDCCI_CHECK_COMM_OPCODE	    0x11
+#define DDCCI_CHECK_TARGET_VALUE    0x90
+#define DDCCI_GET_VERSION_OPCODE    0x99
+#define DDCCI_SET_VERSION_OPCODE    0xBB
```

Comment:
I think these should move to the .rs file, e.g. 

    enum FuRtsHubDdcciOpcode {
        GetVersion = 0x99,
    }